cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/task/LoadMonitorTaskRunner.java

```diff
@@ -45,6 +45,7 @@
 
   private AtomicReference<LoadMonitorTaskRunnerState> _state;
   private volatile double _bootstrapProgress;
+  private volatile boolean _awaitExecution;
```

Comment:
`_awaitExecution` -> `_awaitingPauseSampling`.

Note: I feel like keeping track of an "awaiting pause sampling request" is what we actually want. Such a request can come from 2 sources:
1) Executor can send this command upon starting an execution.
2) The user of Cruise Control can send it via a REST API call to `pauseSampling`.

In either case, we would like to ensure that an ongoing sampling process would know about such a pending request.